template/en/default/reports/report-table.html.tmpl

```diff
@@ -47,34 +50,33 @@
 [% END %]
 
 <script [% script_nonce FILTER none %]>
-function bz_encode (str, decode) {
+function bz_encode (str) {
   if (typeof str !== 'string') {
     str = String(str);
   }
 
-  // First decode HTML entities, if requested.
-  if (decode)
-    str = str.replace(/&lt;/g, "<").replace(/&gt;/g, ">").replace(/&quot;/g, '"')
-             .replace(/&nbsp;/g, " ").replace(/&amp;/g, "&").replace(/\s+$/,"");
+  // First decode HTML entities.
+  str = str.replace(/&lt;/g, "<").replace(/&gt;/g, ">").replace(/&quot;/g, '"')
+           .replace(/&nbsp;/g, " ").replace(/&amp;/g, "&").replace(/\s+$/,"");
 
   // encodeURIComponent() doesn't escape single quotes.
   return encodeURIComponent(str).replace(/'/g, escape);
 };
 
 window.addEventListener('DOMContentLoaded', () => {
-  const linkify = ({ value, data }) => {
+  const linkify = ({ $column, value, data }) => {
```

Comment:
Nit: I think this can just be `column` without the `$`, right?

Reply:
> Nit: I think this can just be `column` without the `$`, right?

You are correct. Technically $column and column are no different to Javascript but in style of JS used in Bugzlla, variables with $ are used to denote a DOM element so I used it to be consistent.